server/leader.go

```diff
@@ -318,6 +333,21 @@ func (s *Server) watchLeader(leader *pdpb.Member, revision int64) {
 	// If the revision is compacted, will meet required revision has been compacted error.
 	// In this case, use the compact revision to re-watch the key.
 	for {
+		//follower watch config change and apply it
+		configHash, err := s.kv.LoadConfigHash()
```

Comment:
This logic will not be executed periodically or immediately. the `rch` will block until the leader changed or some error meet.